Sources/tbDEX/Protocol/Models/Messages/RFQ.swift

```diff
@@ -25,10 +29,59 @@ extension RFQ {
             externalID: externalID,
             protocol: `protocol`
         )
-        self.data = data
-        self.private = nil
     }
+}
 
+private func generateSalt(_ count: Int) throws -> String? {
+    let randomBytes = [UInt8](repeating: UInt8.random(in: 0...255), count: count)
```

Comment:
we're using `SecureRandom` in kotlin, could we use something equivalent to that here? 

i'm seeing [SecureRandomCopyBytes](https://developer.apple.com/documentation/security/1399291-secrandomcopybytes), maybe there's something there